pysd/py_backend/builder.py

```diff
@@ -88,7 +88,7 @@ def build(elements, subscript_dict, namespace, outfile_name):
     if outfile_name == 'return':
         return text
 
-    with open(outfile_name, 'w') as out:
+    with open(outfile_name, 'w', encoding='UTF-8') as out:
```

Comment:
This changes is not affect files with default asci encoding?

Reply:
It shouldn't, as this is internal PySD file writing and reading anyways (PySD should be the only one to read this, apart from a Python interpreter, and the latter can read UTF 8 in any case). If a model has only ascii characters, these will save with no problem as UTF 8 (and Vensim model files are already saved as UTF 8 by default).